app/views/patients/_patient_dashboard.html.erb

```diff
@@ -1,3 +1,12 @@
+<div class="row">
```

Comment:
Can you stick this below the `Appointment date` input? Probably you can take out L2 and L8, and just copy-paste the rest of it onto L28 (so it's a new row below the appointment date div. We have a little bit of real estate to use over there.